runtime/calamari/src/assets_config.rs

```diff
@@ -180,6 +180,7 @@ impl pallet_asset_manager::Config for Runtime {
     type Location = AssetLocation;
     type AssetConfig = CalamariAssetConfig;
     type ModifierOrigin = EnsureRoot<AccountId>;
+    type SuspenderOrigin = EnsureRootOrThreeFourthsCouncil;
```

Comment:
Does it mean no one can call that asset outgoing extrinsic on calamari?

Reply:
No, this is the origin of who can add assets to the filter.